course/flow.py

```diff
@@ -1046,25 +1046,58 @@ def view_flow_page(pctx, flow_session_id, ordinal):
                     or flow_permission.change_answer
                     in permissions)
 
+            # {{{ duplicate code for make_feedback_form
```

Comment:
I introduced a special method to generate in-form feedback for InlineMultipleQuestion, to show which answer is correct and which is not, and here I used duplicate code (NOT DRY), because I find it huge work to reorganize the logic.